### PR TITLE
559 improve wallet modal

### PR DESCRIPTION
**Pull request by brossetti1**
The current v2 is still using the old modal designs for the `UserMenu` Wallet component. This should be updated to match the `react-select` version seen in the v2 `NetworkSwitcher`.

## Description
- updated the alignment of `react-select` menus and added a hover effect on pr - https://github.com/Kwenta/kwenta/pull/584
- add svg assets from ticket
- move WalletOptionsModal into `shared/AppLayout/Header` similar to the file location of `NetworkSwitcher`; renamed `WalletOptionsModal.tsx` to `WalletActions.tsx`
- convert `WalletActions.tsx` to a `react-select` component.
- update en.json - capitalize descriptions of dropdown items.
- made small changes on how the ensName is set and added `truncatedWalletAddress` as a dependency on the `useEffect`

## Related issue
https://github.com/Kwenta/kwenta/issues/559

## Motivation and Context
the wallet actions dropdown should match the styling and feel of the v2 UI using `react-select` - similar to the `NetworkSwitcher`

## How Has This Been Tested?
- [x] testing on different networks
- [x] testing by switching wallets on metamask itself and seeing WalletActions update accordingly with address or ENS name.
- [x] tested using each button on the dropdown to `Change Account`, `Disconnect Account`, and, when connected to a ledger, I tested `Switch Account`
- [x] connected hardware wallet to confirm that the `Switch Account` updates with ledger connections.

## Screenshots (if appropriate):

Screen Shot 2022-04-01 at 11 09 06 AM

Screen Shot 2022-04-01 at 10 26 17 AM

### CODE REVIEW

File: sections/shared/Layout/AppLayout/Header/WalletActions.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+	useEffect(() => {
+		if (signer) {
+			signer.getAddress().then((account: string) => {
+				const _account = account;
+				getENSName(_account, staticMainnetProvider).then((_ensName: string) => {
+					setEns(_ensName);
+				});
+			});
+		}
+	}, [signer, truncatedWalletAddress]);
```
Comment (brossetti1) [inline sections/shared/Layout/AppLayout/Header/WalletActions.tsx:111]: moved this from `UserMenu` into the `WalletActions themselves`

I was noticing some wallet switching issues ENS names and regular wallets. When you have a wallet thats connected to ENS, then switch to a wallet that does not have an ENS, it was failing to update the wallet address. I narrowed this down to two issues:

1) there used to be a [conditional for `setEns(_ensName)`](https://github.com/Kwenta/kwenta/pull/583/files?diff=unified&w=0#diff-040621a7608ebb6a9f7ad541ee07248aab281c740eadac16d6d92189a246bc22L41) - removing this conditional allowed the ens name to be reset to null, allowing the `walletDisplay` value to update correctly.
2) the useEffect was ignoring the `truncatedWalletAddress` as change indicator - adding this to the dependencies retriggers the setEns and subsequentially the walletDisplay value.

File: sections/shared/Layout/HomeLayout/Header.tsx [hunk 2]
```diff
@@ -88,7 +88,7 @@ const Header: FC = () => {
 						))}
 					</Links>
 					<MenuContainer>
-						<UserMenu isTextButton={true} />
+						<UserMenu />
```
Comment (brossetti1) [inline sections/shared/Layout/HomeLayout/Header.tsx:91]: removing this unused `isTextButton` prop

**Comment by brossetti1** [comment]
we might want to make sure that this change does not effect the v1 trading markets

**Review by JChiaramonte7 — DISMISSED** [review]
Thanks @brossetti1 . LGTM 🙂

Can you merge this into a local branch off `v2-dev` first that way we can run CI? If that passes you can merge that into `v2-dev`

**Pull request merged**